Include/internal/pycore_instruction_sequence.h

```diff
@@ -0,0 +1,62 @@
+#ifndef Py_INTERNAL_INSTRUCTION_SEQUENCE_H
+#define Py_INTERNAL_INSTRUCTION_SEQUENCE_H
+
+#ifndef Py_BUILD_CORE
+#  error "this header requires Py_BUILD_CORE define"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+    int h_label;
+    int h_startdepth;
+    int h_preserve_lasti;
+} _PyExceptHandlerInfo;
+
+typedef struct {
+    PyObject_HEAD
```

Comment:
I don't this this is correct. The instructions are allocated as part of an array, so cannot be independently referenced.
```suggestion
```

Reply:
Yeah I forgot to remove this.